schema.cc

```diff
@@ -102,21 +102,30 @@ std::ostream& operator<<(std::ostream& os, ordinal_column_id id)
     return os << static_cast<column_count_type>(id);
 }
 
-std::unique_ptr<dht::i_partitioner> default_partitioner;
+std::map<std::tuple<sstring, unsigned, unsigned>, std::unique_ptr<dht::i_partitioner>> cached_partitioners;
```

Comment:
Missing a comment about why it is safe for this to *not* be `thread_local` and how is access orchestrated.

Reply:
It is actually not safe to not be thread_local and I'm changing this. Thanks @denesb and @avikivity for pointing it out.